<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPlainTextEdit" name="Recv_line_edit">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>611</width>
      <height>421</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="Send_line_edit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>871</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="select_file_path_lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>871</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="select_file_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="read_data_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>读取数据</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>540</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mesg_label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
      <hintingpreference>PreferNoHinting</hintingpreference>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>251</width>
      <height>238</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>串口选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Cbox_port">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>40</number>
       </property>
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="Cbox_boudrate">
         <property name="currentText">
          <string/>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="Cbox_stopbit">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="Cbox_databit">
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>校验位</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="Cbox_parity">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="open_Button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="logo_display_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="select_sn_Box">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>    SN烧录</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="select_token_Box">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>510</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>    Token</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="hex_send_Box">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>510</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> HEX发送</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear_send_edit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>510</y>
      <width>89</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Send_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="Clear_recv_edit">
       <property name="text">
        <string>清空接收</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="hex_recv_Box">
       <property name="text">
        <string> HEX接收</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="save_data_Box">
       <property name="text">
        <string> 自动保存</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="time_Box">
       <property name="text">
        <string> 时间戳</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string> 列数</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="Cbox_data_column">
       <property name="currentText">
        <string>1</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string> 行数</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="Cbox_data_row">
       <property name="currentText">
        <string extracomment="Device Name">1</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="select_Box">
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>      行列数选择</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSN">
    <property name="title">
     <string>SN码烧录工具V1.1</string>
    </property>
   </widget>
   <addaction name="menuSN"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
